### DISCUSSION TITLE
Use of Source in Shortcuts

### DISCUSSION THREAD

**Discussion by m90att** [kalkih/mini-media-player#709] [Q&A]
Hi,  I'm struggling to understand the use and the example of source in the shortcuts...

Definition:
"A source/(sound mode) name can also be specified to change source/(sound mode), use together with type source/sound_mode. "

Example:

https://github.com/kalkih/mini-media-player/blob/master/README.md#card-with-media-shortcuts:~:text=power_state%3A%20false-,Card%20with%20media%20shortcuts,-You%20can%20specify

```
      # Change the source to bathroom
      - icon: mdi:dog
        type: source
        id: Bathroom
```

Can someone tell me what this does and what the example is doing please?  I don't understand what Bathroom is.  

Thanks, Matt

**Comment by mrgrlscz** [comment]
This type of button will change the source of your media_player entity, in this case `bathroom` is the source's name. It's like calling this
```
action: media_player.select_source
data:
  source: bathroom
target:
  entity_id: media_player.YOUR_TV
```
Here's another example for my tv:
```
- icon: mdi:netflix
  type: source
  id: Netflix
```
